Makefile

```diff
@@ -212,7 +212,8 @@ spcc_internal: $(SPCC_DEPS)
 # --randomly-dont-reorganize is to prevent Pytest from shuffling the tests' order
 # --randomly-dont-reset-seed is to make sure that, if we run the same test several times (with
 # @pytest.mark.repeat(3)), different seeds are used, even if things are still deterministic using 
-# the main seed
+# the main seed. --randomly-dont-reset-seed is also important to make that all seeds are different,
```

Comment:
there's already a comment for `--randomly-dont-reset-seed` 2 lines above, so you can remove this

Reply:
the full sentence is 

```
# --randomly-dont-reset-seed is to make sure that, if we run the same test several times (with
# @pytest.mark.repeat(3)), different seeds are used, even if things are still deterministic using 
# the main seed. --randomly-dont-reset-seed is also important to make that all seeds are different,
# from one test to the other
```

I don't think it's a duplicate, it explains several things

Reply:
well then I'm not sure what you are referring to with your new comment, in what circumstances does `--randomly-dont-reset-seed` makes seeds different "from one test to the other" ? maybe elaborate a bit more in the comment

also I think you are missing a `sure` in `important to make [sure] that all seeds` !

Reply:
new text